projects/backend/src/utils/checks/test/check-test.utils.ts

```diff
@@ -0,0 +1,6 @@
+export function assertValidatorErrors(errors: Map<string, string[]>, paramName: string, message: string): void {
+    expect(errors.size).toBe(1);
+    expect(errors.has(paramName)).toBe(true);
+    expect(errors.get(paramName).length).toBe(1);
+    expect(errors.get(paramName)[0]).toBe(message);
```

Comment:
нужно ожидать `undefined`, иначе получим stacktrace при невалидном значении

```suggestion
    expect(errors.get(paramName)?.[0]).toBe(message);
```

Reply:
Я же строчками выше проверяю что значение есть и оно одно. Зачем тут ? добавлять. Это надо быть очень злым чтобы так сделать.

Reply:
так оно дальше пойдет проверки проходить же и получим стектрейс в консоль, не?

Reply:
Нет. После первой ошибки останавливаемся:
![image](https://github.com/Palus-Somni-Team/shikicinema-server/assets/31983783/64db69fe-aa53-4785-906e-b2e7923e142d)